ios/flipswitch_switch/Resources/Info.plist

```diff
@@ -23,7 +23,7 @@
 	<key>DTPlatformName</key>
 	<string>iphoneos</string>
 	<key>MinimumOSVersion</key>
-	<string>3.0</string>
+	<string>7.0</string>
```

Comment:
Why increase the minimum iOS version required? I didn't see Flipswitch dropping support for it in the changelogs

Reply:
Because target deployment version is set to 7.0 - do you want me to change all to 5.0?

Reply:
Ah I see. I added the default to every template to be consistent. We can have it as is for now